SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialInterestConstraints.cpp

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/SpatialInterestConstraints.h"
+
+#include "Interop/SpatialClassInfoManager.h"
+#include "Schema/Interest.h"
+#include "Schema/StandardLibrary.h"
+#include "SpatialConstants.h"
+#include "UObject/UObjectIterator.h"
+
+void UOrConstraint::CreateConstraint(const USpatialClassInfoManager& ClassInfoManager, SpatialGDK::QueryConstraint& OutConstraint) const
+{
+	for (const UAbstractQueryConstraint* ConstraintData : Constraints)
+	{
+		if (ConstraintData)
```

Comment:
We usually are explicit about null checks
```suggestion
		if (ConstraintData != nullptr)
```